packages/fxa-payments-server/src/lib/errors.ts

```diff
@@ -16,8 +16,7 @@ const AuthServerErrno = {
  * - handle Payment token not valid
  */
 
-const BASIC_ERROR =
-  "Hmm, we're having trouble with our system. We're working on fixing it for you and apologize for the inconvenience. Please try again later.";
```

Comment:
FWIW, this was a message reviewed & approved by legal. Might be fine to change, but may be an issue?

Reply:
👍 Thanks for this context. @johngruen sorry for so many @ mentions, but you think we should run this by anyone first?

Reply:
yeah, getting legal sign off ASAP

Reply:
Legal is cool